charts/loki-simple-scalable/templates/read/statefulset-read.yaml

```diff
@@ -38,6 +38,8 @@ spec:
       {{- include "loki.readPriorityClassName" . | nindent 6 }}
       securityContext:
         {{- toYaml .Values.loki.podSecurityContext | nindent 8 }}
+      initContainers:
```

Comment:
WDYT of wrapping this in a `with` block so the key is only included when init containers are provided?